### PR TITLE
Clean up text and fix container registry dashboard

### PR DESCRIPTION
**Pull request by narph**
## What does this PR do?

- consistent naming in category list
- fix container_registry dashboard, azure has updated the metrics 
- fix vmss dashboard and config

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/master/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] If I'm introducing a new feature, I have modified the Kibana version constraint in my package's `manifest.yml` file to point to the latest Elastic stack release (e.g. `^7.13.0`).

## Screenshots

image

### CODE REVIEW

File: packages/azure_metrics/data_stream/compute_vm_scaleset/agent/stream/stream.yml.hbs [hunk 1]
```diff
@@ -82,7 +82,7 @@ resources:
 {{/if}}
 {{#unless resource_ids }}
     {{#unless resource_groups }}
-        - resource_query: "resourceType eq 'Microsoft.Compute/virtualMachines'"
+        - resource_query: "resourceType eq 'Microsoft.Compute/virtualMachineScaleSets'"
```
Comment (jsoriano) [inline packages/azure_metrics/data_stream/compute_vm_scaleset/agent/stream/stream.yml.hbs:85]: Is this a different fix that should be mentioned in the changelog?

File: packages/azure_metrics/kibana/dashboard/azure_metrics-72fe31e7-ea7d-4796-94d1-8f0a091904e2.json [hunk 2]
```diff
@@ -3,157 +3,56 @@
...
         "hits": 0,
         "kibanaSavedObjectMeta": {
-            "searchSourceJSON": {
-                "filter": [],
-                "query": {
-                    "language": "kuery",
-                    "query": ""
-                }
-            }
+            "searchSourceJSON": "{\"query\":{\"language\":\"kuery\",\"query\":\"\"},\"filter\":[]}"
```
Comment (jsoriano) [inline packages/azure_metrics/kibana/dashboard/azure_metrics-72fe31e7-ea7d-4796-94d1-8f0a091904e2.json:6]: Umm, interesting that the JSON objects are not decoded anymore, have you used `elastic-package` to export the dashboards?

File: packages/azure_metrics/data_stream/compute_vm_scaleset/agent/stream/stream.yml.hbs (hunk 1 above)
Comment (narph) [reply packages/azure_metrics/data_stream/compute_vm_scaleset/agent/stream/stream.yml.hbs:85]: it's part of the cleanup process of the metrics package, I have added a mention in the PR description

File: packages/azure_metrics/kibana/dashboard/azure_metrics-72fe31e7-ea7d-4796-94d1-8f0a091904e2.json (hunk 2 above)
Comment (narph) [reply packages/azure_metrics/kibana/dashboard/azure_metrics-72fe31e7-ea7d-4796-94d1-8f0a091904e2.json:6]: exported from kibana

**Review by jsoriano — APPROVED** [review]
Thanks for re-exporting the dashboards!

**Pull request merged**